apps/studio/components/layouts/RealtimeLayout/RealtimeMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+
+import { useParams } from 'common'
+import { useCheckPermissions } from 'hooks'
+import { Edit } from 'lucide-react'
+import Link from 'next/link'
+import { Button, Menu } from 'ui'
+import { CreateChannelModal } from './CreateChannelModal'
+
+export const RealtimeMenu = () => {
+  const router = useRouter()
+  const { ref } = useParams()
+
+  const [showCreateChannelModal, setShowCreateChannelModal] = useState(false)
+  // service api key is required for creating a new channel
+  const canCreateChannels = useCheckPermissions(PermissionAction.READ, 'service_api_keys')
+
+  const page = router.pathname.split('/')[4] as undefined | 'policies' | 'inspector'
+
+  return (
+    <>
+      <Menu type="pills" className="my-6 flex flex-grow flex-col px-5">
+        <div className="mb-6 px-2">
```

Comment:
as an example of why components matter to team:

this `Menu` code has been repeated so many times across the app. There is random tailwind in here,  `my-6 flex flex-grow flex-col px-5` and  `mb-6 px-2` which is now in every layout, of every section. 

It is now such a pain to update the side panel or update it.